app/views/galleries/index.atom.builder

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+def entry_summary(gallery)
+  image_count = gallery.images.count
+  unless gallery.images.count.zero?
+    document = @documents.detect { |doc| doc.fetch(:id, nil) == gallery.images.first.europeana_record_id }
+    if document
+      presenter = Document::SearchResultPresenter.new(document, controller)
+      image_url = presenter.field_value('edmPreview')
+    end
+  end
+  image_url ||= @hero_image && @hero_image.file.present? ? @hero_image.file.url : nil
+
+  %[
+    <h2>#{gallery.title} <span>(#{image_count} #{t('global.labels.images')})</span></h2>
+    <img src=\"#{image_url}\"/>
+    <p>#{gallery.description}<p>
+  ]
+end
+
+atom_feed do |feed|
```

Comment:
To get caching working on the Atom feeds, wrap this `atom_feed` block in:
```ruby
# @todo only insert "views/" into our cache keys when it's needed, i.e. in the context of Mustache view classes
cache(cache_key(@body_cache_key).sub(/\Aviews\//, ''), skip_digest: true) do
  # ...
end
```

We shouldn't really need to do this substitution on the cache key, but we are inserting "views/" into the cache key for the Mustache view classes, whereas the Rails cache helper inserts it itself in the context of view templates like this Atom builder... so we get it repeated. We will want to refactor this when time permits to intelligently insert "views/" only when needed, but for now, the above workaround should suffice here.